das-dashboard/backend/services/config_services.py

```
@@ -1,101 +1,163 @@
+        return {"targets": self.web_config.map_hosts(nested)}
+
+    async def export_config_scp(
+        self,
+        ip: str,
+        configuration_entries: ConfigurationEntriesDto | None = None,
+    ) -> dict:
+        nested = self._build_export_config(configuration_entries)
+        known_ips = {target["ip"] for target in self.web_config.map_hosts(nested)}
+
+        if ip not in known_ips:
+            raise ValueError(f"IP '{ip}' is not configured in the current architecture.")
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)